Automattic/wp-calypso
# Project: Remove arg forwarding "--" from yarn commands

#### Changes proposed in this Pull Request

Using the `yarn run script -- --forwarded-arg` style with yarn isn't needed, and  will output a warning saying the "--" is deprecated.

The warning looks like:
```
warning From Yarn 1.0 onwards, scripts don't require "--" for options to be forwarded. In a future version, any explicit "--" will be forwarded as-is to the scripts.
```

Removing the "--" silences the warning and fixes the issue that in the future the "--" will start getting forwarded to the script too, which could break things.

#### Testing instructions



* The modified scripts should still work and no longer output the above warning
* The testing and typechecking scripts used to output the warning near the start of execution, so easy to test
* The `yarn start` script used to output the warning right after the CSS was built (which is displayed as green text in the console).
